### PR TITLE
tracers ci: drop duktape engine (#24934) and add linux-arm binaries to releases page

### PR DESCRIPTION
**Pull request by j75689**
### Description

This PR aims to remove duktape engine dependency, so that we can support cross-compile on arm

1. ci: add linux-arm binaries.
2. docker: fix dependency issue in Dockerfile
3. eth/tracers: drop duktape engine

### Rationale

N/A

### Example

image

### Changes

Notable changes: 
* Dockerfile
* Makefile
* Ci scripts
* Tracers

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -26,9 +26,9 @@ ENV BSC_HOME=/bsc
 ENV HOME=${BSC_HOME}
 ENV DATA_DIR=/data
 
-ENV PACKAGES ca-certificates~=20220614 jq~=1.6 \
-  bash~=5.1.16-r2 bind-tools~=9.16.29-r0 tini~=0.19.0 \
-  grep~=3.7 curl~=7.83.1-r2 sed~=4.8-r0  curl~=7.83
+ENV PACKAGES ca-certificates~=20220614-r0 jq~=1.6 \
+  bash~=5.1.16-r2 bind-tools~=9.16.33-r0 tini~=0.19.0 \
+  grep~=3.7 curl==7.83.1-r3 sed~=4.8-r0
```
Comment (guagualvcha) [inline Dockerfile:31]: for the `curl`, please do not point a must version, please use `~=`

File: .github/workflows/pre-release.yml [hunk 2]
```diff
@@ -46,6 +45,18 @@ jobs:
...
+      # ==============================
+
+      - name: Build Binary for ARM
+        if: matrix.os == 'ubuntu-18.04'
+        env:
+          GOPATH: /home/runner/work/woodpecker/go
+        run: |
+          mkdir -p $GOPATH/src/github.com/binance-chain/bsc/
+          cp -r ./* $GOPATH/src/github.com/binance-chain/bsc/
+          cd $GOPATH/src/github.com/binance-chain/bsc/ && make geth-linux-arm
```
Comment (brilliant-lx) [inline .github/workflows/pre-release.yml:58]: no binance-chain any more?

File: go.mod [hunk 3]
```diff
@@ -1,14 +1,10 @@
 module github.com/ethereum/go-ethereum
 
-go 1.15
+go 1.17
```
Comment (guagualvcha) [inline go.mod:3]: Please run `go tidy`

File: Dockerfile (hunk 1 above)
Comment (j75689) [reply Dockerfile]: fixed

File: .github/workflows/pre-release.yml (hunk 2 above)
Comment (j75689) [reply .github/workflows/pre-release.yml]: fixed

File: go.mod (hunk 3 above)
Comment (j75689) [reply go.mod:3]: already updated the `go.mod` and `go.sum`

**Review by brilliant-lx — APPROVED** [review]
LGTM

**Review by yutianwu — APPROVED** [review]

**Review by guagualvcha — APPROVED** [review]

**Pull request merged**